app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -0,0 +1,40 @@
+  {{#showSignUpMarketing}}
+    <aside class="marketing-ios" data-marketing-type="android-ios-sync">
+      {{#isiOS}}
+        <p>
+          {{#t}}Get the full Sync experience.{{/t}}
+          <br>
+          {{#t}}Download Firefox for iOS.{{/t}}
+        </p>
+        <a class="marketing-link marketing-link-ios" href="{{ appStoreLinkDirect }}"><img src="{{ appStoreImage }}"></a>
```

Comment:
I know I'm being pedantic - can you add an alt attribute to all of the img tags? Might as well use the same text as used in just after the `<br>`, e.g., `Download Firefox for iOS.`